### PR TITLE
fix Windows Explorer file dialogue navigation commands

### PR DESCRIPTION
**Pull request by alexboche**
Previous navigation commands seem to be off by one field. (also change a-d -> c-l and f6 -> tab)

### CODE REVIEW

**Comment by kendonB** [comment]
Is using `c-l` and `tab` obviously better than using `a-d` and `f6`? Are there circumstances where one works and one doesn't?

File: castervoice/apps/file_dialogue.py [hunk 1]
```diff
@@ -20,12 +20,22 @@ class FileDialogueRule(MergeRule):
...
-            R(Key("a-d, f6:3"), rdescript="File Dialogue: Files list"),
-        "navigation [pane]":
-            R(Key("a-d, f6:2"), rdescript="File Dialogue: Navigation pane"),
+        "search [<text>]":
+            R(Key("c-l, tab") + Text("%(text)s"), rdescript="File Dialogue: Search"),
+        "organize":
+            R(Key("c-l, tab:2"), rdescript="File Dialogue: press 'organize' button"),
+        "(left | navigation) pane":
+            R(Key("c-l, tab:3"), rdescript="File dialogue: navigation pane"),
+        "(center|file|files|folder) (list | pane)":
```
Comment (kendonB) [inline castervoice/apps/file_dialogue.py]: does change this to `((center|file|files|folder) (list | pane)) | files` work?

Comment (kendonB) [reply castervoice/apps/file_dialogue.py]: then you could delete the line below

**Comment by seekM** [comment]
I'm using a german version of Windows 10 and `c-l` takes me to the address bar, `a-d` does nothing (also see [here](https://superuser.com/a/185972/333182)). `tab` works fine for me (can cycle through everything), `F6` lets me switch to the file name input bar only, but not further.

**Comment by LexiconCode** [comment]
Could we use dragonflies ContactAction as a solution?

**Comment by kendonB** [comment]
@seekM, so that we have it documented, can you check what causes `a-d` not to work on your system? If you switch to a US keyboard does it work or is it actually the use of the German language pack?

We might have to test against the system at some stage if one works but not the other on different systems.

**Comment by kendonB** [comment]
"search text" currently bugging out caster
```
Ignoring application 'file_dialogue'. Failed to load with: 
<type 'exceptions.Exception'>
Unknown reference name 'search [<text>]' in 'search [<text>]'
<traceback object at 0x12F32F08>
  File "C:\Users\kmbel\Documents\caster\castervoice\apps\__init__.py", line 13, in is_valid
    _ = __import__(module, globals(), locals())

  File "C:\Users\kmbel\Documents\caster\castervoice\apps\file_dialogue.py", line 58, in <module>
    rule = FileDialogueRule()

  File "C:\Users\kmbel\Documents\caster\castervoice\lib\dfplus\merge\mergerule.py", line 65, in __init__
    MappingRule.__init__(self, name, mapping, extras, defaults, exported)

  File "C:\Python27\lib\site-packages\dragonfly\grammar\rule_mapping.py", line 147, in __init__
    c = Compound(spec, elements=self._extras, value=value)

  File "C:\Python27\lib\site-packages\dragonfly\grammar\elements_compound.py", line 280, in __init__
    element = self._parser.parse(spec)

  File "C:\Python27\lib\site-packages\dragonfly\parser.py", line 59, in parse
    return node.value()

  File "C:\Python27\lib\site-packages\dragonfly\parser.py", line 306, in value
    return self.actor.value(self)

  File "C:\Python27\lib\site-packages\dragonfly\grammar\elements_compound.py", line 67, in value
    repetitions = [node.children[0].value()]

  File "C:\Python27\lib\site-packages\dragonfly\parser.py", line 306, in value
    return self.actor.value(self)

  File "C:\Python27\lib\site-packages\dragonfly\parser.py", line 516, in value
    return [c.value() for c in node.children]

  File "C:\Python27\lib\site-packages\dragonfly\parser.py", line 306, in value
    return self.actor.value(self)

  File "C:\Python27\lib\site-packages\dragonfly\grammar\elements_compound.py", line 124, in value
    element = alternative.children[0].value()

  File "C:\Python27\lib\site-packages\dragonfly\parser.py", line 306, in value
    return self.actor.value(self)

  File "C:\Python27\lib\site-packages\dragonfly\grammar\elements_compound.py", line 213, in value
    child = node.children[1].value()

  File "C:\Python27\lib\site-packages\dragonfly\parser.py", line 306, in value
    return self.actor.value(self)

  File "C:\Python27\lib\site-packages\dragonfly\grammar\elements_compound.py", line 67, in value
    repetitions = [node.children[0].value()]

  File "C:\Python27\lib\site-packages\dragonfly\parser.py", line 306, in value
    return self.actor.value(self)

  File "C:\Python27\lib\site-packages\dragonfly\parser.py", line 516, in value
    return [c.value() for c in node.children]

  File "C:\Python27\lib\site-packages\dragonfly\parser.py", line 306, in value
    return self.actor.value(self)

  File "C:\Python27\lib\site-packages\dragonfly\grammar\elements_compound.py", line 124, in value
    element = alternative.children[0].value()

  File "C:\Python27\lib\site-packages\dragonfly\parser.py", line 306, in value
    return self.actor.value(self)

  File "C:\Python27\lib\site-packages\dragonfly\grammar\elements_compound.py", line 157, in value
    raise Exception(s)

Caster: Dragonfly is up-to-date
```

**Comment by LexiconCode** [comment]
Outside of meeting the fix the merge conflicts what is the status of the pull request?

**Comment by alexboche** [comment]
I'd say it's done. I tested this when I made the last commit though that was a while ago.

**Comment by kendonB** [comment]
@alexboche can you not replicate the "search text" error I was seeing with the PR?

**Comment by kendonB** [comment]
Still need to fix conflicts too

**Comment by alexboche** [comment]
Yes the problem with the "search" command was fixed with the previous commit, sorry for not being clear about that. I just merged with the latest caster so I think this is finished.

**Comment by LexiconCode** [comment]
Nice work! Alex! It's ready to emerge from my perspective was anyone else has feedback.

**Comment by LexiconCode** [comment]
@alexboche I've resolve the conflicts removing the rdescript as well. If you want to do a quick test and take a peek then I will merge.

**Comment by alexboche** [comment]
okay I just tested a couple command seems to be working

**Pull request merged**